frr_exporter.go

```diff
@@ -77,15 +58,15 @@ func main() {
 		}
 		landingPage, err := web.NewLandingPage(landingConfig)
 		if err != nil {
-			logger.Error(err.Error())
+			logger.Error("Couldn't create landing page", "err", err.Error())
 			os.Exit(1)
 		}
 		http.Handle("/", landingPage)
 	}
 
 	server := &http.Server{}
 	if err := web.ListenAndServe(server, webFlagConfig, logger); err != nil {
-		logger.Error(err.Error())
+		logger.Error("Error starting HTTP server", "err", err.Error())
```

Comment:
I have followed the style of node_exporter with this and the landing page error, where the do not explicitly add the error as a key-val, rather just blurt it out as the message itself.